pldoc/_includes/footer-site.html

```diff
@@ -0,0 +1,5 @@
+<footer class="pldoc-footer-site">
```

Comment:
Is this includes used anywhere?  I did a quick search and didn't see it referenced.